frontend/src/metabase/querying/notebook/components/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -203,29 +200,24 @@ function LegacyDataPicker({
   stageIndex,
   table,
   placeholder,
-  hasMetrics,
   isDisabled,
   onChange,
 }: LegacyDataPickerProps) {
-  const databaseId = Lib.databaseID(query);
   const tableInfo =
     table != null ? Lib.displayInfo(query, stageIndex, table) : undefined;
+  /**
+   * We try to blur the line between models and tables for embedding users.
+   * this property will change the way icons are displayed in the data picker trigger,
+   * so we need to remove it. Treating it as a table.
+   */
+  delete tableInfo?.isModel;
   const pickerInfo = table != null ? Lib.pickerInfo(query, table) : undefined;
-  const { data: card } = useGetCardQuery(
```

Comment:
remove unused props after removing props below

Reply:
Ah, thanks! I'll take a look at this 🙏 . There's probably no tests for this as I don't see the interactive embedding tests crashes yet (well I just fixed the whole suite)

Reply:
Fixed